packages/terra-clinical-result/tests/jest/ClinicalResultBloodPressure.test.jsx

```diff
@@ -9,6 +9,8 @@ import {
   NoDataResult,
 } from '../../src/terra-dev-site/test/clinical-result/TestResults';
 
+jest.mock('uuid/v4', () => () => '55b406c0-057c-4e82-94fb-85a2fb87c29c');
```

Comment:
There should be a test to see if uuid is being set. We can also use `00000000-0000-0000-0000-000000000000` as the mock value for better maintainability.

Reply:
Added tests here - https://github.com/cerner/terra-clinical/pull/868/commits/f932e12eaad49881c42630811d1619d1000807ed#diff-f78e39f6ea1427cf52ee7390619f232d70376b20b4e2739af9e9f78b662cc681R32 and updated to use `00000000-0000-0000-0000-000000000000` here - https://github.com/cerner/terra-clinical/pull/868/commits/f932e12eaad49881c42630811d1619d1000807ed#diff-f78e39f6ea1427cf52ee7390619f232d70376b20b4e2739af9e9f78b662cc681R13